infer/tests/codetoanalyze/java/dependencies/.inferconfig

```diff
@@ -0,0 +1,42 @@
+{
+    "biabduction": false,
+    "fragment-retains-view": false,
+    "inefficient-keyset-iterator": false,
+    "starvation": false,
+    "racerd": false,
+    "eradicate": false,
+    "pulse": true,
+    "report-block-list-path-regex": [
+        ".class$"
```

Comment:
@jvillard Is this the recommended way to avoid reporting on every file with the extension `.class` ? It does not seem to be working.

Reply:
@jeremydubreil I think it's because the regex has to match the whole file path so you want `".*\.class$"`